src/Components/Cards/ProductCard/ProductCard.css

```diff
@@ -1,6 +1,26 @@
-/* .card {
-    width: 20vw;
-    max-width: 400px;
-    min-width: 300px;
-    margin-bottom: 5rem;
-} */
\ No newline at end of file
+.card {
```

Comment:
hey rohit, you can add border to card component so that it looks neat to user's eye

Reply:
Added a shadow on hover, looks great now thanks for the suggestion